lib/schema_validations/active_record/validations.rb

```diff
@@ -202,7 +204,7 @@ def _filter_validation(macro, name) #:nodoc:
           types << match[1].to_sym
         end
         return false if config.only        and not Array.wrap(config.only).include?(name)
-        return false if config.except      and     Array.wrap(config.except).include?(name)
+        return false if config.whitelist      and     Array.wrap(config.whitelist).include?(name)
         return false if config.only_type   and not (Array.wrap(config.only_type) & types).any?
         return false if config.except_type and     (Array.wrap(config.except_type) & types).any?
```

Comment:
Do you think it's better to change `except_type`  to `whitelist_type` as well? Just to be consistent.